WordPress/src/main/java/org/wordpress/android/util/FluxCUtils.java

```diff
@@ -54,6 +55,8 @@ public static MediaFile fromMediaModel(MediaModel media) {
         mediaFile.setDescription(media.getDescription());
         mediaFile.setCaption(media.getCaption());
         mediaFile.setUploadState(media.getUploadState());
+        mediaFile.setVideo(media.getMimeType().startsWith("video"));
+        mediaFile.setVideoPressShortCode(ShortcodeUtils.getVideoPressShortcodeFromId(media.getVideoPressGuid()));
```

Comment:
I know this is kind of stupid, especially because we're doing the exact opposite in the `EditorFragment`, we should just drop the `MediaFile` completely at some point.